src/Components/README.md

```diff
@@ -61,10 +73,23 @@ source activate.sh
 . ./activate.ps1
 ```
 
-4. Now you can build `src\Components\Web.JS\` by running the following commands in the `src\Components\Web.JS` directory:
+1. Now you can build all the JavaScript assets required by the repository (including SignalR for instance) by running the following command:
+
+```powershell
+npm run build
+```
+
+1. Build the Components:
+
+```powershell
+cd src/Components
+./build.cmd
+```
+
+2. Optionally, open the Components in Visual Studio:
 
 ```powershell
-dotnet build
+.\startvs.cmd
```

Comment:
Should it repeat `cd src/Components` since it was done in the previous step?

Reply:
An alternative would be to make both stateless by running, e.g. `src\Components\build.cmd`.